sktime/forecasting/enbpi.py

```diff
@@ -128,13 +134,34 @@ def __init__(
 
         super().__init__()
 
-        from tsbootstrap import MovingBlockBootstrap
+        if bootstrap_transformer.get_tag("object_type") == "bootstrap":
+            self.bootstrap_transformer_ = TSBootstrapAdapter(
+                bootstrap_transformer, return_indices=True
+            )
+        else:
+            self.bootstrap_transformer_ = bootstrap_transformer
+
+        if self.bootstrap_transformer is None:
+            warn(
+                "The default value for the bootstrap_transformer will change to the"
+                + "sktime MovingBlockBootstrap in version 0.37.0."
+                + "If you want to use the current behavior"
```

Comment:
Clearer perhaps: "For the current default behaviour after 0.37.0, pass bootstrap_transformer= ..."

Reply:
Done